requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+# formatting
```

Comment:
Can we try to merge this into [pyproject.toml](https://github.com/lm-sys/FastChat/blob/main/pyproject.toml)?